catalog/security/banner-marking/src/main/java/org/codice/alliance/security/banner/marking/BannerMarkings.java

```diff
@@ -319,7 +319,7 @@ public String toString() {
 
     private List<String> jointAuthorities;
 
-    private List<String> usFgiCountryCodes = null;
+    private List<String> usFgiCountryCodes;
```

Comment:
❓  Is this change sufficient?

Reply:
I couldn't really see another way to fix it. I wanted to talk to @shaundmorris about it before we merged.

Reply:
❓ @josephthweatt did you run this with SonarLint and see if it resolved it?

Reply:
@shaundmorris yes, and the issue was resolved when I got rid of the null. However, I wanted to see if this was the right way of resolving these issues.

Reply:
I would be okay with it. But would like @coyotesqrl's thought on it since he wrote the code. My understanding is that an empty list would be serializable here, but a null list is not.

Reply:
That should be fine, @josephthweatt. It doesn't really matter from a workflow perspective, as all the collections run through the `ensureCollectionInitialized` method during the parsing process. If this silences the static analysis warning, that's great.